CorsixTH/Lua/languages/english.lua

```diff
@@ -828,6 +828,7 @@ cheats_window = {
     toggle_epidemic = "Toggle epidemics",
     epidemic = "Spawn contagious patient",
     show_infected = "Show or hide infected icons",
+    toggle_infected = "Show or hide infected icons",
```

Comment:
Move this and 857 to the unused section. You might want to follow the comment block in 953-956 but I'll leave that up to you.
Do the strings actually need to exist to not crash? Only instance I could think where they might crash is during the afterload sequence if the `UICheats` window was open.

Reply:
I think they are needed in the moment before the window is closed.

> You might want to follow the comment block in 953-956

Good idea, done.